### ISSUE TITLE
TemplateData messages

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#328]
Looking at de.wikipedia.org/wiki/Vorlage:Lang/Doku in xowa I get
![templatedata1](https://user-images.githubusercontent.com/861631/51071854-d26b8500-164f-11e9-97d2-72f5244628cf.jpg)
I tracked this down to missing messages - found in mediawiki\extensions\TemplateData\i18n are language files with messages
taking a subset of messages for german (de.json) and english (en.json) [messages starting with 'templatedata-doc-param']
I added these to de.gfs and en.gfs respectively (with appropriate transliteration)
Then I get
![templatedata2](https://user-images.githubusercontent.com/861631/51071927-f11e4b80-1650-11e9-86f6-c5d996865cdb.jpg)

I suppose I should include all the messages?
Is there a limit?

There is also one final small issue - looking at the real page
![templatedata3](https://user-images.githubusercontent.com/861631/51071984-a4874000-1651-11e9-8c7b-5be30ccf451a.jpg)

**Comment by desb42** [comment]
Oops - press Comment when I intended to add more
The red ovals show that `Vorlagenparameter` should be in the table of contents but it is not in the xowa version

**Comment by gnosygnu** [comment]
Yeah, I think this is the 3rd or so bug regarding un-updated messages. It's something I want to do, so you don't have to do it yourself. I'm tagging this one as "language" and will come back to pick this up later

Regarding your other question: There's no practical limit. The language file gets loaded when you open a related wiki. It should weigh in at less than a few MB in memory. I don't think I need to add any type of dynamic loading / caching yet, but I guess we'll see.

Finally, the last ToC issue is a bug, though at the moment, I don't know how. XOWA generates the ToC after all the HTML has been built, so it should be there. I'll debug later.

Thanks.
